server/index.js

```diff
@@ -28,12 +27,11 @@ app.get('/businesses/:businessId/images', (req, res) => {
 });
 
 app.post('/businesses/:businessId/images', (req, res) => {
-  // console.log(req.body);
   dbHelp.postImage(
     { id: req.params.businessId, body: req.body },
     (err, data) => {
-      if (err) res.status(400).send(err);
-      else res.send('SENT');
+      if (err) res.json(err);
+      else res.json('SENT');
```

Comment:
Do you plan to send a status code too?

Reply:
express.json sends an automatically generated status code, but you're right I'll send a status for the error messages.